app/components/textarea/textarea.html.erb

```diff
@@ -1,5 +1,23 @@
-<div <%= attrs.slice(:class) %>>
```

Comment:
The `Textarea` component is larger and more complex over time, including many features that might not be needed in all cases, for example:

* Autosizing depending on the length of the content
* Displaying shortcut hints
* Highlighting placeholders

I’m not sure if it’s a good idea to keep adding functionality to this component. On the other hand, it seemed to be the simplest solution to implement this.